src/PrestaShopBundle/Translation/Provider/BackofficeCatalogueProvider.php

```diff
@@ -0,0 +1,115 @@
+<?php
+/**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ */
+declare(strict_types=1);
+
+namespace PrestaShopBundle\Translation\Provider;
+
+use PrestaShopBundle\Translation\Loader\DatabaseTranslationLoader;
+use Symfony\Component\Translation\MessageCatalogue;
+
+class BackofficeCatalogueProvider implements CatalogueProviderInterface
+{
+    /**
+     * @var DatabaseTranslationLoader
```

Comment:
The BackofficeCatalog is fetched from the database so we need to connect to ~SGBD~

As we aim to build something a lot better than the current implementation, and hopefully a lot easier to understand and maintain, let's make it very easy to understand 😄

Reply:
Be aware that "SGBD" is French

Reply:
You're right, the English translation is DBMS 👍 
```suggestion
     * The BackofficeCatalog is fetched from the database so we need to connect to DBMS
     * @var DatabaseTranslationLoader
```